packages/elements-core/src/context/Options.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import type { DocsProps } from '../components/Docs';
+
+const DEFAULT_CONTEXT: ElementsOptionsContextProps = {};
+
+export type ElementsOptionsContextProps = Pick<DocsProps, 'nodeHasChanged'>;
+
+export const ElementsOptionsContext = React.createContext<ElementsOptionsContextProps | undefined>(undefined);
+
+export const useOptionsCtx = () => {
+  return React.useContext(ElementsOptionsContext) || DEFAULT_CONTEXT;
```

Comment:
```suggestion
  return React.useContext(ElementsOptionsContext);
```

Reply:
wouldn't that work ^?

Reply:
This is to cover the "just in case" the context provider was not used to wrap the area.

Reply:
Then, the default would still be used no?

Reply:
assuming it's set here https://github.com/stoplightio/elements/pull/2218#discussion_r971883500

Reply:
No if code calls `useOptionsCtx` and the tree has not been wrapped in the context provider, `React.useContext(ElementsOptionsContext)` will return undefined/null.